test/e2e/specs/editor/various/undo.spec.js

```diff
@@ -455,6 +455,35 @@ test.describe( 'undo', () => {
 			},
 		] );
 	} );
+
+	// @see https://github.com/WordPress/gutenberg/issues/12075
+	test( 'should be able to undo and redo property cross property changes', async ( {
+		page,
+		pageUtils,
+	} ) => {
+		await page.getByRole( 'textbox', { name: 'Add title' } ).type( 'a' ); // First step.
+		await page.keyboard.press( 'Backspace' ); // Second step.
```

Comment:
Could we also add a test for this for blocks? The title case is a bit different? Maybe we need some logic in RichText that treats Backspace changes differently from input, which normally only marks changes as persistent after 1s.

https://github.com/WordPress/gutenberg/blob/2b687d03ecdfb52229c25d5d455038b14fc8fa89/packages/block-editor/src/components/rich-text/use-mark-persistent.js#L12

Reply:
Can you give me an example of test you have in mind. The issue fixed by this PR is hardly reproducible with blocks in trunk.